ntbs-service/Pages/Shared/_NotificationEditLayout.cshtml

```diff
@@ -0,0 +1,14 @@
+@model ntbs_service.Pages.Notifications.NotificationModelBase
+
+@{
+    Layout = "./_NotificationLayoutWithAlertsAndSideMenu.cshtml";
+}
+
+@section MainFormButton {
+    <div hidden>
```

Comment:
Is this a taghelper from somewhere?

Reply:
Ah, it's just an attribute - and it successfully defaults to hidden="hidden" in render, so no complaints here